src/components/Layout/NavigationLayout/GenreNavigation/GenreNavigation.tsx

```diff
@@ -22,16 +39,33 @@ const GenreNavigation = (): JSX.Element => {
         {option.id}
       </button>
     ));
+
+  function updateFilters(newFilters): void {
+    queryParams.delete(FILTER_BY);
```

Comment:
`set` can be used instead of delete + append

Reply:
sure